src/components/ErrorNotification/ErrorNotification.tsx

```
@@ -0,0 +1,18 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { FC } from 'react';
+
+type Props = {
+  errorMessage: string
+};
+
+export const ErrorNotification: FC<Props> = ({ errorMessage }) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className="notification is-danger is-light has-text-weight-normal"
+    >
+      <button data-cy="HideErrorButton" type="button" className="delete" />
```

When you click on this button you should hide the error(remove errorMessage)